nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/service/TestStandardControllerServiceProvider.java

```diff
@@ -218,7 +221,7 @@ public void testStartStopReferencingComponents() {
             public Object answer(InvocationOnMock invocation) throws Throwable {
                 final ProcessorNode procNode = (ProcessorNode) invocation.getArguments()[0];
                 procNode.verifyCanStart();
-                procNode.setScheduledState(ScheduledState.RUNNING);
+                // procNode.setScheduledState(ScheduledState.RUNNING);
```

Comment:
True, and I think this test(s) may need to be removed since tests that I added as part of the refactoring efforts cover all possible scenarios without Mocks. And if for some reason you'll find a missing scenario, let me know and I'll add a test.